src/styles/pages/productdetail/product-info.scss

```diff
@@ -13,7 +13,7 @@ a.product-title {
   font-size: $font-size-corporate;
   color: $text-color-quaternary;
 
-  label {
+  .product-id-name {
```

Comment:
rename to product-id-label

Reply:
renamed the class from `product-id-name` to `product-id-label`